alicloud/resource_alicloud_db_account.go

```diff
@@ -63,7 +75,24 @@ func resourceAlicloudDBAccountCreate(d *schema.ResourceData, meta interface{}) e
 	request.RegionId = client.RegionId
 	request.DBInstanceId = d.Get("instance_id").(string)
 	request.AccountName = d.Get("name").(string)
-	request.AccountPassword = d.Get("password").(string)
+
+	password := d.Get("password").(string)
+	kmsPassword := d.Get("kms_encrypted_password").(string)
+
+	if password == "" && kmsPassword == "" {
+		return WrapError(Error("you have to specify one of 'password' and 'kms_encrypted_password fields!"))
```

Comment:
`		return WrapError(Error("you have to specify one of 'password' and 'kms_encrypted_password fields!"))
`
to
`
		return WrapError(Error("One of the  'password' and 'kms_encrypted_password' should be set."))
`